wsgi_kerberos.py

```diff
@@ -3,14 +3,25 @@
 
 Add Kerberos/GSSAPI Negotiate Authentication support to any WSGI Application
 '''
+import errno
```

Comment:
`_consume_request()` below [contains the line](https://github.com/twosigmajab/wsgi-kerberos/commit/d0a6f41fa48dadf1e3628eb4577f775cc4bbf630#diff-2c47e40846c837f0f98f71d933bfcf3fR59)                  
```python
if err.errno != errno.EAGAIN:
```
which would fail with a NameError without this line. If you'd prefer this be separated out into a dedicated PR with a test case (since the current tests didn't catch this), let me know and I'll take this out of this PR.

Reply:
Good catch, thanks.